content/intro/7.md

```diff
@@ -1,13 +1,20 @@
 +++
-date = "2017-04-26T22:29:00+10:00"
-prev = "/intro/6"
-next = "/basic/1"
-title = "Congratulations"
-weight = 6
+date = "2017-04-26T22:28:58+10:00"
+next = "/intro/6"
+prev = "/intro/8"
```

Comment:
These two should be swapped, or else there's a cycle between 6 and 7 when navigating to the next page.

Reply:
done